pkg/sensors/tracing/kprobe_test.go

```diff
@@ -6580,3 +6581,249 @@ spec:
 	err = jsonchecker.JsonTestCheck(t, checker)
 	assert.NoError(t, err)
 }
+
+func getArmKprobeSymb(kSymbols *ksyms.Ksyms) string {
+	if kSymbols.IsAvailable("arm_kprobe") == true {
+		return "arm_kprobe"
+	} else if kSymbols.IsAvailable("arm_kprobe_ftrace") == true {
+		// Are we optimized on top of arch ftrace?
+		return "arm_kprobe_ftrace"
+	} else if kSymbols.IsAvailable("__arm_kprobe") == true {
+		return "__arm_kprobe"
+	} else if kSymbols.IsAvailable("arch_arm_kprobe") == true {
+		// fallback to arch specific
+		return "arch_arm_kprobe"
+	}
+	return ""
+}
+
+func getDisarmKprobeSymb(kSymbols *ksyms.Ksyms) string {
+	if kSymbols.IsAvailable("disarm_kprobe") == true {
+		return "disarm_kprobe"
+	} else if kSymbols.IsAvailable("disarm_kprobe_ftrace") == true {
+		// Are we optimized on top of arch ftrace?
+		return "disarm_kprobe_ftrace"
+	} else if kSymbols.IsAvailable("__disarm_kprobe") == true {
+		return "__disarm_kprobe"
+	} else if kSymbols.IsAvailable("arch_disarm_kprobe") == true {
+		// fallback to arch specific
+		return "arch_disarm_kprobe"
+	}
+	return ""
+}
+
+func TestKprobeTracing(t *testing.T) {
+	var doneWG, readyWG sync.WaitGroup
+	defer doneWG.Wait()
+
+	ctx, cancel := context.WithTimeout(context.Background(), tus.Conf().CmdWaitTime)
+	defer cancel()
+
+	kSymbols, err := ksyms.KernelSymbols()
+	if err != nil {
+		t.Skipf("Skipping test could not fetch kernel symbols: %v", err)
+	}
+
+	arm_kprobe := getArmKprobeSymb(kSymbols)
+	if arm_kprobe == "" {
+		t.Skipf("Skipping test could not get appropriate symbol for tracing arm kprobes")
+	}
+
+	disarm_kprobe := getDisarmKprobeSymb(kSymbols)
+	if disarm_kprobe == "" {
+		t.Skipf("Skipping test could not get appropriate symbol for tracing disarm kprobes")
+	}
+
+	tracingPolicy := `
+apiVersion: cilium.io/v1alpha1
+kind: TracingPolicy
+metadata:
+  name: "kprobes-tracing"
+  annotations:
+    description: "Detects kprobes operations"
+spec:
+  kprobes:
+  - call: ` + arm_kprobe + `
```

Comment:
hum, so this should not work with kprobe multi,  I'd think you need to add option like:

```
options:
- name: "disable-kprobe-multi"
  value: "1"
```